### PR TITLE
Lexicographica

### PR DESCRIPTION
**Pull request by jaw111**
Remove some gerunds

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
Thanks a lot!

File: spec/latest/triple-pattern-fragments/index.html [hunk 1]
```diff
@@ -229,7 +229,7 @@
         <dt>hypermedia controls</dt>
         <dd>
           A Triple Pattern Fragment MUST contain hypermedia controls
-          that allow retrieving any other Triple Pattern Fragment of the same dataset.
+          that allow retrieval of any other Triple Pattern Fragment of the same dataset.
```
Comment (vanthome) [inline spec/latest/triple-pattern-fragments/index.html]: Any reasoning why the continuous tense is wrong here? I learned that it makes documents more lively and is thus to be preferred.

Comment (jaw111) [inline spec/latest/triple-pattern-fragments/index.html]: Depends on writing style. I had learned from Simplified Technical English that use of gerunds is to be avoided as they can cause confusion for non-native readers (and can be awkward to translate). Downside is it leads to a 'drier' style, but if that aids comprehension then it's fine with me.

Comment (vanthome) [inline spec/latest/triple-pattern-fragments/index.html]: ok, thanks for the explanation!

**Pull request merged**